litmus-portal/frontend/src/graphql/queries.ts

```diff
@@ -459,6 +459,12 @@ export const GET_TEMPLATE_BY_ID = gql`
   }
 `;
 
+export const GET_PREDEFINED_EXPERIMENT_LIST = gql`
```

Comment:
If it's for fetching workflows we may rename (similarly in backend) or we use same API atleast in frontend. @amityt